cdc/sink/mq/mq_flush_worker.go

```diff
@@ -270,3 +252,7 @@ func (w *flushWorker) flushAndNotify(ctx context.Context) error {
 
 	return nil
 }
+
+func (w *flushWorker) closeMsgChan() {
```

Comment:
Why not name the method `close` so we can put more finalize code into it?

Reply:
For now we only need to close this one resource, so name it more precisely. If we have a need to extend it, we can rename it in the future.

Reply:
LGTM, even though the name is a little bit ugly.